src/pubtools/_pulp/tasks/delete.py

```diff
@@ -78,7 +79,10 @@ def add_args(self):
 
         self.parser.add_argument(
             "--repo",
-            help="remove content from these comma-seperated repositories ",
+            help="remove content from these comma-seperated repositories. "
+                 "If '%s' is used, the package will be removed from all repos,"
```

Comment:
Looks to be missing a space:

```suggestion
                 "If '%s' is used, the package will be removed from all repos, "
```

Without this it will look not quite right, like this:

`the package will be removed from all repos,excluding all-rpm-content-*`